source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+Organizing Your Sidebar
+=======================
+
+You can organize your sidebar based on how you use Mattermost. Customizations you make to your sidebar only changes your Mattermost sidebar.
+
+Here's how your sidebar is set up by default:
```

Comment:
Here’s how your sidebar is setup by-default:

- All channels you've joined appear in the Channels category, sorted alphabetically.
- All your Direct Messages appear in the Direct Messages category, sorted by recent activity.